tilequeue/command.py

```diff
@@ -1070,6 +1070,50 @@ def tilequeue_dump_tiles_of_interest(cfg, peripherals):
     )
 
 
+def tilequeue_load_tiles_of_interest(cfg, peripherals):
+    """
+    Given a newline-delimited file containing tile coordinates in
+    `zoom/column/row` format, load those tiles into a new Redis SET
+    alongside the existing one, then RENAME the new to the old.
+    """
+    logger = make_logger(cfg, 'load_tiles_of_interest')
+    logger.info('Loading tiles of interest')
+
+    toi_filename = "/Users/iandees/Downloads/toi.txt"
```

Comment:
Yes, ironically we started off that way, but we started moving everything to the config yaml because the command lines got unwieldy and we were repeating a lot most of the time.

Instead of calling `create_command_parser`, I'd suggest making your own version. There's also a little bit in the configuration that tries to merge what would have been passed on the command line with the yaml, but that hasn't been exercised in a bit.